### PR TITLE
Add opts argument to issue query method

### PR DESCRIPTION
**Pull request by kenchan**
Issue Query API can take some parameters. (e.g. https://github.com/treasure-data/td-client-ruby/blob/master/lib/td/client/api/job.rb#L275 )

I'd like to use it. So, I add argument and refactor code.

I would like you to review my code. thanks.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -285,23 +285,34 @@ TDClient.prototype = {
      *  Submit Hive type job
      *  @param {string} db - The name of database
      *  @param {string} query - The Hive query which run on given database
+     *  @param {object} opts - Optoins supported member are `result`,
```
Comment (Lewuathe) [inline lib/index.js]: Typo `Optoins` -> `Options`.
I think `Supported options are ~` might be better.

**Comment by Lewuathe** [comment]
@kenchan Overall, looks very good to me. Thank you so much!

**Comment by kenchan** [comment]
@Lewuathe Thank you for your review!

**Comment by Lewuathe** [comment]
@kenchan Thanks again!

**Pull request merged**